### PR TITLE
AlgerianAdministrativeDivision project added to dzcode.io

### PR DESCRIPTION
**Pull request by mohsenuss91**
# Description
[AlgerianAdministrativeDivision ](https://github.com/mohsenuss91/AlgerianAdministrativeDivision) project added to [dzcode.io](https://github.com/dzcode-io) based on [dzcode.io article](https://www.dzcode.io/Articles/How_To_Add_Your_Project_To_DzCode/).

### CODE REVIEW

**Review by ZibanPirate — CHANGES_REQUESTED** [review]
Thank you @mohsenuss91 for submitting your awesome project, i have two small requests, and your PR will be ready to merge:
- Thanks for the link fixes, but please revert back the last two commits, cuz they are not related to submitting your project, and if you'd like, create a seperate PR for them and will merge it right away.
- run `yarn lint:fix` to fix pipeline error, then commit the changes

**Comment by mohsenuss91** [comment]
> Thank you @mohsenuss91 for submitting your awesome project, i have two small requests, and your PR will be ready to merge:
> 
> * Thanks for the link fixes, but please revert back the last two commits then force push, cuz they are not related to submitting your project, and if you'd like, create a seperate PR for them and will merge it right away.
> * run `yarn lint:fix` to fix pipeline error, then commit the changes

* Two commits reverted.
* but when I run `yarn lint:fix`  I've got this errors:
`$ yarn lint:fix
yarn run v1.22.5
$ concurrently "cd fullstack && yarn eslint:fix && yarn prettier:fix" "cd data &
& yarn eslint:fix && yarn prettier:fix" "cd frontend && yarn eslint:fix && yarn
prettier:fix"
'concurrently' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this comm
and.
`

**Comment by ZibanPirate** [comment]
i see, it's fine @mohsenuss91, just run these commands step by step, and it should work fine:

1 - revert the last 4 commits, until you reach the first commit, do that by running this command **4 times**
```sh
git reset HEAD~
```
2 - clear your changes
```sh
git reset --hard
```
3 - make sure to install all dependencies
```sh
yarn
```
4 - fix linting errors
```sh
yarn lint:fix
```
5 - commit your changes
```sh
git add *
git commit -m "fixed linting errors"
```
6 - finally, force push your changes
```sh
git push -f
```

**Comment by mohsenuss91** [comment]
> i see, it's fine @mohsenuss91, just run these commands step by step, and it should work fine:
> 
> 1 - revert the last 4 commits, until you reach the first commit, do that by running this command **4 times**
> 
> ```shell
> git reset HEAD~
> ```
> 
> 2 - clear your changes
> 
> ```shell
> git reset --hard
> ```
> 
> 3 - make sure to install all dependencies
> 
> ```shell
> yarn
> ```
> 
> 4 - fix linting errors
> 
> ```shell
> yarn lint:fix
> ```
> 
> 5 - commit your changes
> 
> ```shell
> git add *
> git commit -m "fixed linting errors"
> ```
> 
> 6 - finally, force push your changes
> 
> ```shell
> git push -f
> ```

@ZibanPirate Done. Thanks for your help!

File: data/projects/Algerian_Administrative_Division/info.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "title": "AlgerianAdministrativeDivision",
```
Comment (ZibanPirate) [inline data/projects/Algerian_Administrative_Division/info.json:2]:
```suggestion
  "title": "Algerian Administrative Division",
```

File: data/projects/Algerian_Administrative_Division/info.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "description": "Administrative division of Algeria (Wilayas - Daïras - Baladiyahs)",
+  "githubURI": "https://github.com/mohsenuss91/AlgerianAdministrativeDivision",
```
Comment (ZibanPirate) [inline data/projects/Algerian_Administrative_Division/info.json:4]:
```suggestion
  "githubURI": "mohsenuss91/AlgerianAdministrativeDivision",
```

File: data/projects/Algerian_Administrative_Division/info.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  "slug": "Algerian_Administrative_Division",
+  "image": "https://github.com/mohsenuss91/AlgerianAdministrativeDivision/blob/master/assets/images/AlgerianAdministrativeDivision%20logo%20svg.svg"
```
Comment (ZibanPirate) [inline data/projects/Algerian_Administrative_Division/info.json:6]:
```suggestion
  "image": "https://raw.githubusercontent.com/mohsenuss91/AlgerianAdministrativeDivision/master/assets/images/AlgerianAdministrativeDivision%20logo%20svg.svg"
```

File: data/projects/list.json [hunk 4]
```diff
@@ -1,4 +1,11 @@
 {
-  "items": ["Leblad", "Mishkal", "React_Project_Builder", "Kuliya", "DzCode"],
+  "items": [
+    "Leblad",
+    "Mishkal",
+    "React_Project_Builder",
+    "Kuliya",
+    "DzCode",
+    "AlgerianAdministrativeDivision"
```
Comment (ZibanPirate) [inline data/projects/list.json:8]:
```suggestion
    "Algerian_Administrative_Division"
```

File: data/projects/ssr.json [hunk 5]
```diff
@@ -1,4 +1,11 @@
 {
-  "items": ["Leblad", "Mishkal", "React_Project_Builder", "Kuliya", "DzCode"],
+  "items": [
+    "Leblad",
+    "Mishkal",
+    "React_Project_Builder",
+    "Kuliya",
+    "DzCode",
+    "AlgerianAdministrativeDivision"
```
Comment (ZibanPirate) [inline data/projects/ssr.json:8]:
```suggestion
    "Algerian_Administrative_Division"
```

**Review by ZibanPirate — CHANGES_REQUESTED** [review]
That's great, one last change request and will merge your PR, to properly display your project, please apply the suggested changes i mentioned in the thread

**Comment by mohsenuss91** [comment]
> That's great, one last change request and will merge your PR, to properly display your project, please apply the suggested changes i mentioned in the thread

Done! 👍

**Review by ZibanPirate — APPROVED** [review]
Great, thanks for submitting your project! It will be in the website in the next release!

**Comment by mohsenuss91** [comment]
@ZibanPirate Thanks for your help! 👍🏻

**Comment by mohsenuss91** [comment]
> Great, thanks for submitting your project! It will be in the website in the next release!

Hey @ZibanPirate! 👋
The project still doesn't show upin dzcode.io website! 😔

**Comment by ZibanPirate** [comment]
Hi @mohsenuss91 , the project is only added to [dzcode.io/Projects](https://www.dzcode.io/Projects), i created a PR ( #238 ) to add it also on the landing page, thanks for notice!

**Comment by mohsenuss91** [comment]
@ZibanPirate  Thanks for your help!

**Pull request merged**